external-crates/move/crates/move-compiler/src/typing/core.rs

```diff
@@ -616,17 +616,27 @@ impl<'env> Context<'env> {
 // Subst
 //**************************************************************************************************
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum RefKind {
+    Value,
```

Comment:
After playing with this a bit, I think we should do `Owned` here. Since references are also values :)